### PR TITLE
Added option flipvertical

### PR DESCRIPTION
**Pull request by kirkau**
Rotates a label to the correct orientation when line runs east-> west.

Needs to be integrated with direction check (via bearing)

### CODE REVIEW

**Comment by leplatrem** [comment]
Thanks for submitting your PR !

Could you please : 

* Fix the indentation
* Mention the new option in the README file

Thanks !

File: leaflet.textpath.js [hunk 1]
```diff
@@ -120,6 +120,12 @@ var PolylineTextPath = {
             textNode.setAttribute('dx', ((pathWidth / 2) - (textWidth / 2)));
         }
 
+        if (options.flipvertical) {   
+			var rotatecenterX = (textNode.getBBox().x + textNode.getBBox().width / 2);
+			var rotatecenterY = (textNode.getBBox().y + textNode.getBBox().height / 2);			
+			textNode.setAttribute('transform', 'rotate(90 ' + rotatecenterX + ' ' + rotatecenterY + ')');
```
Comment (leplatrem) [inline leaflet.textpath.js]: It's not 180 ?

**Comment by kirkau** [comment]
Great... will do some more work on it and re submit after I've had a chance to test properly

**Comment by fredericbonifas** [comment]
Thanks, you PR is merged!

**Pull request merged**